fido/src/main/java/com/yubico/yubikit/fido/client/BasicWebAuthnClient.java

```diff
@@ -601,6 +592,27 @@ private boolean getCtapUv(String userVerification, boolean pinProvided) throws C
         }
     }
 
+    /**
+     * Calculates the preferred pinUvAuth protocol version for authenticator provided list.
+     * Returns PinUvAuthDummyProtocol.VERSION if the authenticator does not support any of the SDK
+     * supported protocols.
+     */
+    private int getPreferredPinUvAuthProtocol(List<Integer> pinUvAuthProtocols) {
```

Comment:
Would it make more sense to return an instance of PinUvAuthProtocol?

Reply:
Good suggestion, I will update.

Reply:
fixed